src/main/java/org/apache/commons/lang3/time/DateUtils.java

```diff
@@ -1831,4 +1831,40 @@ public void remove() {
         }
     }
 
+    /**
+     * Obtain full standalone month names as used in "LLLL" date formatting.
+     * @param locale Locale
+     * @return Long names of months
+     */
+    static String[] getStandaloneLongMonths(final Locale locale) {
+        // Unfortunately standalone month names are not available in DateFormatSymbols,
+        // so we have to extract them.
+        final Calendar calendar = Calendar.getInstance(locale);
+        final int monthCount = calendar.getMaximum(Calendar.MONTH) + 1;
+        final String[] monthNames = new String[monthCount];
+        for (int i = 0; i < monthCount; i++) {
+            calendar.set(Calendar.MONTH, i);
+            monthNames[i] = calendar.getDisplayName(Calendar.MONTH, Calendar.LONG_STANDALONE, locale);
+        }
+        return monthNames;
+    }
+
+    /**
+     * Obtain short standalone month names as used in "LLLL" date formatting.
+     * @param locale Locale
+     * @return Short names of months
+     */
+    static String[] getStandaloneShortMonths(final Locale locale) {
+        // Unfortunately standalone month names are not available in DateFormatSymbols,
+        // so we have to extract them.
+        final Calendar calendar = Calendar.getInstance(locale);
+        final int monthCount = calendar.getMaximum(Calendar.MONTH) + 1;
+        final String[] monthNames = new String[monthCount];
+        for (int i = 0; i < monthCount; i++) {
+            calendar.set(Calendar.MONTH, i);
+            monthNames[i] = calendar.getDisplayName(Calendar.MONTH, Calendar.SHORT_STANDALONE, locale);
+        }
+        return monthNames;
+    }
```

Comment:
These two methods are identical except for Calendar.SHORT_STANDALONE vs. Calendar.LONG_STANDALONE? Why duplicate? Javadoc: "Obtain..." -> "Gets..."

Reply:
sorry, I really should have noticed that before submitting. I'll refactor

Reply:
refactored into one method and changed javadoc. While doing this I looked at the Calendar source and Calendar#getDisplayNames seems to be more efficient than using the Calendar#getDisplayName method multiple times, so I am now using that.